var/spack/repos/builtin/packages/libgrpp/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libgrpp(CMakePackage):
+    """A library for the evaluation of molecular integrals of the"""
+
+    """generalized relativistic pseudopotential operator (GRPP) over Gaussian functions."""
+
+    homepage = "https://github.com/aoleynichenko/libgrpp"
+    git = "https://github.com/aoleynichenko/libgrpp.git"
+
```

Comment:
If possible, it'd obviously be great to have a tagged first release here, but I udnerstand that may not be possible. This is not a blocker.